force-app/main/default/lwc/systemTools/systemTools.html

```diff
@@ -0,0 +1,9 @@
+<template>
+    <div class="slds-grid--vertical slds-theme_default">
```

Comment:
is there a reason why this is not using saving canvas?

Reply:
There is no edit and save/cancel for this page @xinnanlisf